File: TsogosunProfileAdmin/MSPatronDetails/Model/Dtos/ApprovedUserDetailsDto.cs
using System;

namespace tsogosun.com.MSPatronDetails.Model.Dtos
{
    public class ApprovedUserDetailsDto
    {
        public int SiteId { get; set; }
        public long UserId { get; set; }
        public string IdpassportNO { get; set; }
        public string MobileNumber { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int? Rating { get; set; }
        public string RecordStatus { get; set; }
        public DateTime? DateConfirmed { get; set; }
        public string ConfirmedBy { get; set; }
    }
}

File: TsogosunProfileAdminWeb/src/app/model/Dtos/ms-patron-details/approvedUserDetailsDto.ts
export interface ApprovedUserDetailsDto {
    siteId: number;
    userId: number;
    idpassportNO?: string | undefined;
    mobileNumber?: string | undefined;
    firstName?: string | undefined;
    lastName?: string | undefined;
    rating?: number | undefined;
    recordStatus?: string | undefined;
    dateConfirmed?: Date | undefined;
    confirmedBy?: string | undefined;
}